profile/README.md

```diff
@@ -3,3 +3,66 @@
 `karmor profile` which shows real-time terminal user interface table of three different operations going on in KubeArmor: Process, File and Network. It maintains a counter of each operation that is happening within the cluster, along with other useful details. It directly fetches data from the `karmor logs` API and displays all the required information. The TUI includes simple navigation between operations and a user input based filter as well.
```

Comment:
> operations going on in KubeArmor: Process, File and Network

please improve this message,
operations are not related to KubeArmor, instead observed by KubeArmor.